README.md

```diff
@@ -135,9 +135,9 @@ because we didn't assign it to anything. Don't worry about that.
 Here's another:
 
 ```js
-const b = stampit().init(function () {
+const b = stampit().init(({instance}) => {
```

Comment:
This example should be working. I would rather leave it as is to showcase that every initializer is bound to `this`. Could you please revert this line and the change above back to initial state?
Thank you